pandas/core/config.py

```diff
@@ -53,8 +53,10 @@
 import re
 import warnings
 
-import pandas.compat as compat
-from pandas.compat import lmap, map, u
+try:
```

Comment:
Assuming this is targeted for 0.25 do we even need to bother with this?

Reply:
I figure until we actually start ripping out compat code (any day now I hope) its easier to leave this in and keep the ball rolling on these

Reply:
We could just wait until `0.24.2` comes out before merging this.